pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -90,12 +97,13 @@
     }
 
     @GET
-    @Path("/{cluster}/{broker}/ownedNamespaces")
+    @Path("/{clusterName}/{broker-webserviceurl}/ownedNamespaces")
```

Comment:
Seems other rest endpoints also use cluster/broker for short, Is here a special need to change this?

Reply:
I think `broker-webservice` is necessary. Firstly I think `broker` is broker service url. So I think if using `broker-webservcie` is more clear.